variant_remapping_tools/reads_to_remapped_variants.py

```diff
@@ -64,32 +75,60 @@ def is_read_valid(read, counter, flank_length, score_cutoff, diff_cutoff):
     return True
 
 
-def process_bam_file(bam_file_path, output_file, old_ref_allele_output_file, flank_length, score_perc, diff_AS_XS_perc):
+def fetch_bases(fasta, contig, length, start):
+    """
+    Returns a subsection from a specified FASTA contig. The start coordinate is 1-based.
+    """
+    zero_base_start = start - 1
+    end = zero_base_start + length
+    new_ref = fasta.fetch(reference=contig, start=zero_base_start, end=end)
+    return new_ref
+
+
+def calculate_new_alleles(old_ref, new_ref, old_alt, is_reverse_strand):
+    """Given allele and strand information, calculate the new (ref, alt) pair."""
+    # If reverse strand -> calculate complement
+    old_ref_processed = old_ref
+    old_alt_processed = old_alt
+    if is_reverse_strand:
+        old_ref_processed = Seq(old_ref_processed, generic_dna).complement()
+        old_alt_processed = Seq(old_alt_processed, generic_dna).complement()
```

Comment:
As Kirill says it should be reverse_complement()
```suggestion
        old_ref_processed = Seq(old_ref_processed, generic_dna).reverse_complement()
        old_alt_processed = Seq(old_alt_processed, generic_dna).reverse_complement()
```